packages/api-v2/src/hooks/p2p/entity/order/p2p-order/useOrderList.ts

```diff
@@ -8,30 +9,53 @@ const useOrderList = (
     config?: NonNullable<Parameters<typeof useInfiniteQuery<'p2p_order_list'>>[1]>['options']
 ) => {
     const { isSuccess } = useAuthorize();
-    const { data, fetchNextPage, ...rest } = useInfiniteQuery('p2p_order_list', {
+    const { data: subscriptionData, subscribe, unsubscribe } = useSubscription('p2p_order_list');
+
+    // Subscribe to the p2p_order_list endpoint to keep track of the order list updates
+    React.useEffect(() => {
+        if (isSuccess) subscribe();
+
+        return () => {
+            unsubscribe();
+        };
+    }, [isSuccess]);
```

Comment:
@ameerul-deriv, just a question, don't you need to add  `subscribe` and `unsubscribe` to the dependancy array here? 🤔 I'm not sure if the linter is complaining or not
```suggestion
    }, [isSuccess]);
```

Reply:
when i did it kept on spamming the subscribe call 😓 so I left it out, and yeah linter will complain